File: UtilitiesVB/To Depricate/StringManipulation.vb
Imports UtilitiesCS
Imports System.Reflection

Public Module StringManipulation
    Public Function GetStrippedText(strTmp As String) As String
        If NotImplementedDialog.StopAtNotImplemented(MethodBase.GetCurrentMethod().Name) Then
            Throw New NotImplementedException
        End If
        Return strTmp
    End Function
End Module

File: UtilitiesCS/To Depricate/StringManipulation.cs
using System;
using System.Reflection;
using System.Text.RegularExpressions;
using UtilitiesCS;

namespace UtilitiesCS
{

    public static class StringManipulation
    {
        public static string GetStrippedText(string text)
        {
            var regex = new Regex("[^\u0020-\u007D]");
            return regex.Replace(text, "");
            
            //if (NotImplementedDialog.StopAtNotImplemented(MethodBase.GetCurrentMethod().Name))
            //{
            //    throw new NotImplementedException();
            //}
            //return strTmp;
        }
    }
}